### PR TITLE
6093-November-ReleaseNotes-Readme

### PR DESCRIPTION
**Pull request by hemanttanwar**
Adding Service level README.md as requested by documentation team.
A lot of text is replicated from root README. 
Fixes https://github.com/Azure/azure-sdk-for-java/issues/6093

### CODE REVIEW

**Comment by hemanttanwar** [comment]
@kaerm Please revire

File: sdk/storage/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+- [Azure Storage Common](azure-storage-common/README.md) provides infrastructure shared by the other Azure Storage client libraries like shared key authentication and exceptions.
+- [Azure Storage File Share](azure-storage-file-share/README.md) service shares are accessible via the Server Message Block (i.e. SMB) protocol, and also via REST APIs. The File Share service offers the following four resources: the storage account, shares, directories, and files. Shares provide a way to organize sets of files and also can be mounted as an SMB file share that is hosted in the cloud.
+- [Azure Storage File Datalake](azure-storage-file-datalake/README.md) is Microsoft's optimized storage solution for for big data analytics workloads. A fundamental part of Data Lake Storage Gen2 is the addition of a hierarchical namespace to Blob storage. The hierarchical namespace organizes objects/files into a hierarchy of directories for efficient data access. 
+
```
Comment (kaerm) [inline sdk/storage/README.md]: Looking good! In the directory I also see Queue Cryptography, Common, management ones, and microsoft-azure-storage-blob? should they be described too?

Comment (hemanttanwar) [reply sdk/storage/README.md]: I have already explained 'Azure Storage Common'  and others you mentioned are not in this folder, what you see is just a copy of old versions. And  we do not need to explain them.

Comment (kaerm) [reply sdk/storage/README.md]: So they are going to be deleted? Maybe I am checking something wrong but V11 is still there?

Comment (hemanttanwar) [reply sdk/storage/README.md]: I need to find out from team why we are keeping 6-7 older version of azure-mgmt-storage in this location. And based on that I will inform you what are we going to do with this.

Comment (kaerm) [reply sdk/storage/README.md]: Thanks! Please do - if they are to stay there we should mention what they are.

**Comment by kaerm** [comment]
@hemanttanwar - also sorry, just noticed you also added KV here - are you planning to add descriptions there later?

**Pull request merged**